frameworks/kafka/notes.md

```diff
@@ -0,0 +1,20 @@
+# Testing a package on a local cluster
```

Comment:
Note to self - remove this from this PR.

Note to self - document how to locally build and run this stuff against a minidcos cluster.